github-email-notifications/config.py

```diff
@@ -1 +1,16 @@
-config = dict(githubconsumer=True)
+config = {
+    "testgithubconsumer": False,  # set to True for enabling consumer
+    "testgithubconsumer.email_to": u"test@example.com",
+    "testgithubconsumer.email_from": u"noreply@example.com",
+    "testgithubconsumer.smtp_server": u"smtp.example.com",
+
+    "sssdgithubconsumer": False,
+    "sssdgithubconsumer.email_to": u"test@example.com",
+    "sssdgithubconsumer.email_from": u"noreply@example.com",
```

Comment:
Should we already set something more sensible like sssd-github-notification@fedorahosted.org ?